### ISSUE TITLE
Cannot query a column with BIT(8) data type

### ISSUE THREAD

**Issue by newpcraft** [CUBRID/cubrid-manager#125]
```
CREATE TABLE bit_tbl(a1 BIT, a2 BIT(1), a3 BIT(8), a4 BIT VARYING);
INSERT INTO bit_tbl VALUES (B'1', B'1', B'1', B'1');
SELECT * FROM bit_tbl;
```

When run above query, it threw an exception like:

```
Exception in thread "Running Query..." java.lang.ClassCastException: java.lang.Boolean cannot be cast to [B
	at com.cubrid.common.ui.spi.util.FieldHandlerUtils.getRsValueForExport(FieldHandlerUtils.java:250)
	at com.cubrid.common.ui.query.control.QueryExecuter.BuildCurrentRowData(QueryExecuter.java:504)
	at com.cubrid.common.ui.query.control.QueryExecuter.fillTableItemData(QueryExecuter.java:611)
	at com.cubrid.common.ui.query.control.QueryExecuter.makeResult(QueryExecuter.java:297)
	at com.cubrid.common.ui.query.editor.QueryEditorPart$QueryThread.run(QueryEditorPart.java:2794)
	at java.lang.Thread.run(Thread.java:745)
```

It only happens on BIT(8) type, so it seems it needs a workaround of fetching data manner from a resultset, because cubrid manager uses ResultSet.getObject() method to fetch a cell data commonly, but due to jdbc issue it throws an exception in some case such as BIT(8) type.

**Comment by newpcraft** [comment]
In FieldHandlerUtils.getRsValueForExport():
```java
Object data = rs.getObject(colNumber);

} else if (DataType.DATATYPE_BIT.equals(colType) || DataType.DATATYPE_BIT_VARYING.equals(colType)) {
  ...
  byte[] bytes = (byte[]) data;
  ...
}
```

It would be better to make it fetches a cell data though `ResultSet.getBytes()` instead of getObject for BIT and BIT varying data types.

**Comment by hun-a** [comment]
Thank you for suggesting a good solution!
We will check and correct it. 😃

**Comment by hun-a** [comment]
@newpcraft 
At the recently CM from source code - Branch of `develop` -, this problem was not reproduced.
<img width="619" alt="result" src="https://user-images.githubusercontent.com/16603187/41753521-e315e356-7608-11e8-9068-393ee781e00b.png">

Could you tell me your version of CM?

**Comment by hun-a** [comment]
We've found that the same problem occurs while fixing issue #136.
As a result, this issue has been fixed in the #136 issue.
If you want to know details, please refer to [here](https://github.com/CUBRID/cubrid-manager/pull/136/commits/0a2b11c570be64303dbcfd3e7ef119636ba99986).

Thank you for your report. 😃
